galicaster/mediapackage/mediapackage.py

```diff
@@ -23,6 +23,8 @@
 import re
 import time
 import os
+import dateutil
```

Comment:
I like the second option more, `dateutil.parser.parse()`, since that's how the `dateutil.tz.tzutz()` is being used in that same line.